docs/deployment.md

```diff
@@ -35,49 +36,62 @@ RelayServer deployments consist of multiple instances (pods) of different contai
 ## RelayServer Components
 
 * Thinktecture.Relay.IdentityServer  
-  The purpose of this component is to provide a default way how a connector can authenticate. A connector uses the
-  OAuth client credential flow with the IdentityServer component to receive an access token for authenticated
-  communication with the RelayServer. The service needs to request a single new token in regular intervals. This
-  component does not need to handle a lot of load, and the load is expected to increase in a linear way with the
-  amount of concurrently running connectors.  
-  
-  **DO** run at least two instances of this container on different machines in order to provide a reliable system.
+  The purpose of this component is to provide a default way how a connector can authenticate. A connector uses the OAuth
+  client credential flow with the IdentityServer component to receive an access token for authenticated communication
+  with the RelayServer. The service needs to request a single new token in regular intervals. This component does not
+  need to handle a lot of load, and the load is expected to increase in a linear way with the amount of concurrently
+  running connectors.
+
+  **DO** create your own project using the RelayServer v3 NuGet packages. Use the corresponding persistence packages for
+  the database of your choice.  
+  **Do NOT** use the example / development project and docker file from this repository in production.  
+  **Do NOT** run multiple instances of the RelayServer.IdentityServer component, as this is not yet supported.
 
   Configuration:
-  * Provide a shared volume to all instances of Thinktecture.Relay.IdentityServer on /var/creds
-  * Provide a connection string to the RelayServer database 
+  * Provide a connection string to the RelayServer database.
 
-* Thinktecture.Relay.Server
+* Thinktecture.Relay.Server  
```

Comment:
Please remove the trailing white spaces.

Reply:
This is correct markdown rendering syntax to enforce a line-break,

With trailing whitespace:
![image](https://user-images.githubusercontent.com/755148/213134499-29d2d784-db4d-496e-82b7-7b620316f87d.png)

Without trailing whitespace:
![image](https://user-images.githubusercontent.com/755148/213134738-206300f2-0382-48c8-a6ef-b6943dcda2a2.png)